### ISSUE TITLE
[New Feature]: Automate keeping package manifest up to date with PowerShell modules Evergreen and Nevergreen

### ISSUE THREAD

**Issue by o-l-a-v** [microsoft/winget-pkgs#34776]
### Description of the new feature/enhancement

To make less management overhead/ manual labour keeping the package manifest up to date, it would be cool if one could use PowerShell modules Evergreen and Nevergreen (and maybe others?) in a scheduled task somewhere (Azure Automation Account?) to suggest updates to ```winget-pkgs```.

Approvement logic could be automated tests (Actions/ DevOps pipeline), then human accept maybe?

Evergreen, by Aaron Parker.
* https://www.powershellgallery.com/packages/Evergreen
* https://stealthpuppy.com/evergreen/
* https://github.com/aaronparker/Evergreen

Nevergreen, by Dan Gough.
* https://www.powershellgallery.com/packages/Nevergreen
* https://github.com/DanGough/Nevergreen

**Comment by Trenly** [comment]
There is already wingetbot, which tries to update sources where it can. @vedantmgoyal2009 has also create [winget-pkgs-automation](/vedantmgoyal2009/winget-pkgs-automation) which is used to update many of the packages in the repository here.

**Comment by vedantmgoyal2009** [comment]
I had already implemented most of the packages in Evergreen in my automation. I was also planning to take from Nevergreen but I am looking for a better method since webpage scraping can break when websites are changed and hence packages will be left outdated. 

Also, I make sure that the source/API is updated timely since I came through some packages which have a source where we can check for updates but the publisher updates the source too late or some releases get missed.
